### DISCUSSION TITLE
Links to websites and channels about cosmology

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#282] [General]
So I mainly use the https://arxiv.org/archive/astro-ph daily feeds (especially cosmology, astrophysics of galaxies and instrumentation) though some other tags about theories (qr) and hep phen and hep ex are interesting too.
I also use https://phys.org/space-news/ which is remarkably exhaustive for a mainstream media, it is highly complementary to arxiv since it covers non open access journals and also news outside of scientific papers (such as from a NASA blog post).

The only arxiv reviewer I know of is https://mbustamante.net/my-daily-arxiv-picks/ he is highly prolific but seems to be partly specialized in particles physics and dark matter.

The most popular blogger specialized in MOND is https://tritonstation.com/
Then there is Kroupa, which is less prolific but has highly complementary research
https://darkmattercrisis.wordpress.com/#:~:text=The%20Cookson%20Controversy

in terms of youtube channels there is Sabine
https://www.youtube.com/@SabineHossenfelder

I get access to closed access pdfs via email request or scihub or libtsc
For the outsiders there is vixra though it's a mixed bag
I have also found some very original ideas on qeio https://www.qeios.com/read/OZ0IRU.2
Feel free to share other recommendations!

todo write about rydberg matter polaron redshift, dih, rri, stability and superdense hydrogen (and proton radius prediction)
https://arxiv.org/abs/2410.21379
https://arxiv.org/abs/2410.21461
https://link.aps.org/accepted/10.1103/PhysRevD.104.062003
https://arxiv.org/abs/2410.22098
https://arxiv.org/abs/2410.21401
https://www.researchgate.net/publication/346831724_Cosmology_astrobiology_and_the_RNA_world_Just_add_quintessential_water
https://arxiv.org/abs/2107.07571
https://arxiv.org/abs/2410.00657
https://arxiv.org/abs/2410.18884
https://arxiv.org/abs/2410.23019

**Comment by RedshiftDrift** [comment]
More sources, some already listed above and others:

- Astrobites https://astrobites.org/
- A search on arXiv that finds articles [with the word "surprising* in the abstract](https://arxiv.org/search/advanced?advanced=&terms-0-operator=AND&terms-0-term=surpris*+cosmolog*&terms-0-field=all&classification-physics=y&classification-physics_archives=astro-ph&classification-include_cross_list=include&date-year=&date-filter_by=date_range&date-from_date=2023-12&date-to_date=2024-12&date-date_type=submitted_date&abstracts=show&size=50&order=-announced_date_first)
- Astrodon https://astrodon.social
- A search on Yandex for *JWST high redshift galaxies* https://yandex.com/search/?text=JWST+high+redshift+galaxy&lr=10115&within=2
- Pavel Kroupa's *Dark Matter Crisis* https://darkmattercrisis.wordpress.com/
- Stacy McGaugh's Triton Station https://tritonstation.com/
- Utrecht Philosophy of Astronomy & Cosmology https://www.uu.nl/en/research/utrecht-philosophy-of-astronomy-cosmology
- A Cosmology Group https://cosmology.info/
- RSS feeds
- - Cosmology talks https://www.youtube.com/@CosmologyTalks/videos
- - Sabine Hossenfelder https://www.youtube.com/@SabineHossenfelder/videos
- - Anton Petrov https://www.youtube.com/@whatdamath/videos
- - Space News https://phys.org/space-news/
- - See the Pattern https://www.youtube.com/@SeethePattern/videos
- - AAS Nova https://aasnova.org/
- - Cosmology news/Space Time https://www.sciencedaily.com/news/space_time/
- - The Open Journal of Astrophysics https://astro.theoj.org/
- - Science Magazine https://www.science.org/news/all-news
- - Eric Lerner's LPPFusion https://www.youtube.com/@LPPFusion/videos
- - Unzicker's Real Physics https://www.youtube.com/@TheMachian/videos
- - Sky & Telescope https://skyandtelescope.org/
- - Cosmology Science https://cosmologyscience.com/cosblog/top-ten-big-bang-myths-talk-for-nz-dunedin-astronomical-society/ (no recent posts)
- Woit's *Not even wrong* https://www.math.columbia.edu/~woit/wordpress/ sometimes addresses issues in cosmology

Hoping everybody will share links to interesting news about cosmology, from these websites and others.

**Reply by LifeIsStrange** [reply]
> A search on arXiv that finds articles [with the word "surprising* in the abstract](https://arxiv.org/search/advanced?advanced=&terms-0-operator=AND&terms-0-term=surpris*+cosmolog*&terms-0-field=all&classification-physics=y&classification-physics_archives=astro-ph&classification-include_cross_list=include&date-year=&date-filter_by=date_range&date-from_date=2023-12&date-to_date=2024-12&date-date_type=submitted_date&abstracts=show&size=50&order=-announced_date_first)

Loves this one!! Thanks for sharing :)

**Comment by RedshiftDrift** [comment]
"JWST’s Deepest Look Shatters the Big Bang Timeline"
https://youtu.be/EOowNVML4_I

**Comment by OlliSant** [comment]
Facebook group:
https://www.facebook.com/share/g/1JVpZNNbMU/

**Comment by RedshiftDrift** [comment]
[The *See the Pattern* Video Archive Is Live!](https://www.patreon.com/posts/see-pattern-is-142343256?utm_campaign=patron_engagement&utm_source=post_link&post_id=142343256&utm_id=c1421432-190d-4288-829b-894845a6e1dd&utm_medium=email)

"This is still a work in progress, so not every video is there yet..."
